spring-integration-cassandra/src/main/resources/org/springframework/integration/config/xml/spring-integration-cassandra-1.0.xsd

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xsd:schema xmlns="http://www.springframework.org/schema/integration/kafka"
-			xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-			xmlns:tool="http://www.springframework.org/schema/tool"
-			xmlns:integration="http://www.springframework.org/schema/integration"
-			targetNamespace="http://www.springframework.org/schema/integration/kafka"
-			elementFormDefault="qualified" attributeFormDefault="unqualified">
-
-	<xsd:import namespace="http://www.springframework.org/schema/beans"/>
-	<xsd:import namespace="http://www.springframework.org/schema/tool"/>
+<xsd:schema
+	xmlns="http://www.springframework.org/schema/integration/cassandra"
+	xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:beans="http://www.springframework.org/schema/beans"
+	xmlns:tool="http://www.springframework.org/schema/tool"
+	xmlns:integration="http://www.springframework.org/schema/integration"
+	targetNamespace="http://www.springframework.org/schema/integration/cassandra"
+	elementFormDefault="qualified" attributeFormDefault="unqualified">
+	<xsd:import namespace="http://www.springframework.org/schema/beans" />
+	<xsd:import namespace="http://www.springframework.org/schema/tool" />
 	<xsd:import namespace="http://www.springframework.org/schema/integration"
-				schemaLocation="http://www.springframework.org/schema/integration/spring-integration.xsd"/>
+		schemaLocation="http://www.springframework.org/schema/integration/spring-integration-4.2.xsd" />
```

Comment:
I really prefer to make extension project version free. And rely on that SI-core version which is in classpath.
It isn't problem that you have moved this project to the SI-4.2.3, but that would be great that we will be compatible with `4.3` in the future without re-compiling here.